homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_3/rec.js

```diff
@@ -0,0 +1,29 @@
+const readline = require('readline');
+
+const inputData = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+inputData.question('Введите минимальное значение: ', (min) => {
+    inputData.close();
+    let a = 1;
+    let b;
+    function rec(n) {
+        if (a > n) {
+            return false;
+        }
+        if (a === n) {
+            return true;
+        }
+        a *= 2;
+        return rec(n);
+    }
+
+    for (let i = min; i <= 600; i++) {
+        b = rec(i);
+        if (b === true) {
+            console.log(i);
+        }
+    }
+});
```

Comment:
again, avoid you functions to interact with data out their scope, unless it's necessary

for the second task, your script should consist of two units

1. recursive function taking one argument
2. loop utilizing this function

no extra variables outside, no side effects